### ISSUE TITLE
typo on privacy policy VO mispronounces it

### ISSUE THREAD

**Issue by JuliannaR** [cds-snc/covid-shield-mobile#494]
There's a typo on Privacy Policy page. "Your name or adress". A 'd' is missing in address so VO mispronounces it
